packages/data-helpers/src/expand-entity/tests/expand-entity.test.ts

```diff
@@ -0,0 +1,44 @@
+import { expandEntity } from '../expand-entity';
+
+describe('ExpandEntity', () => {
+    let entity;
+
+    beforeAll(async () => {
+        entity = {
+            data: {
+                name: 'Kevin Nderitu',
+                age: 12,
+                address: {
+                    location: 'Nairobi'
+                }
+            }
+        };
+    });
+    it('should be a function', async () => {
+        expect(expandEntity).toBeInstanceOf(Function);
+    });
+    it('should convert successfully', async () => {
+        const expected = {
+            data: {
+                name: {
```

Comment:
@grafitto as a suggestion, not necessarily something to implement, you could also check out snapshot testing, it might save you the time of writing this whole `expected` block and having to manually extend it when you add new tests cases, i.e. using `expect(expandEntity(entity)).toMatchSnapshot()`

Reply:
@grafitto what does `expandEntity` do if the entity has other fields besides `data`, like `_id`, `tags`, etc.?

Reply:
Other fields are ignored and returned as they are

Reply:
@habbes is that what was required?

Reply:
I think that's good, so long as they're not removed